src/Drupal/Commands/core/MigrateRunnerCommands.php

```diff
@@ -0,0 +1,706 @@
+                    );
+                }
+
+                $table[] = [
+                    'id' => $printedMigrationId,
+                    'status' => $status,
+                    'total' => $sourceRows,
+                    'imported' => $imported,
+                    'needing_update' => $needingUpdate,
+                    'unprocessed' => $unprocessed,
+                    'last_imported' => $lastImported,
+                ];
+            }
+
+            // Add an empty row after a tag group.
+            if ($tag) {
+                $table[] = $this->prepareTableRow([], $namesOnly);
+            }
+        }
+
+        return new RowsOfFields($table);
+    }
+
+    /**
+     * Prepares a table row for migrate status.
+     *
+     * @param array $row
+     *   The row to be prepared.
+     * @param true|null $namesOnly
+     *   If to output only the migration IDs.
+     *
+     * @return array
+     *   The prepared row.
+     */
+    protected function prepareTableRow(array $row, ?bool $namesOnly): array
+    {
+        if (!$namesOnly) {
+            $row += array_fill_keys([
+              'id',
+              'status',
+              'total',
+              'imported',
+              'needing_update',
+              'unprocessed',
+              'last_imported',
+            ], null);
+        }
+
+        return $row;
+    }
+
+    /**
+     * Perform one or more migration processes.
+     *
+     * @command migrate:import
+     *
+     * @drupal-dependencies migrate
+     *
+     * @param string|null $migrationIds
+     *   Comma-separated list of migration IDs.
+     *
+     * @option all Process all migrations.
+     * @option tag A comma-separated list of migration tags to import
+     * @option limit Limit on the number of items to process in each migration
+     * @option feedback Frequency of progress messages, in items processed
+     * @option idlist Comma-separated list of IDs to import. As an ID may have more than one column, concatenate the columns with the colon ':' separator
+     * @option update In addition to processing unprocessed items from the source, update previously-imported items with the current data
+     * @option force Force an operation to run, even if all dependencies are not satisfied
+     * @option execute-dependencies Execute all dependent migrations first.
+     * @option timestamp Show progress ending timestamp in progress messages
+     * @option total Show total processed item number in progress messages
+     * @option progress Show progress bar
+     *
+     * @usage migrate:import --all
+     *   Perform all migrations
+     * @usage migrate:import --all --no-progress
+     *   Perform all migrations but avoid the progress bar
+     * @usage migrate:import --tag=user,main_content
+     *   Import all migrations tagged with user and main_content tags
+     * @usage migrateimport classification,article
+     *   Import new terms and nodes using migration 'classification' and 'article'
```

Comment:
Words enclosed in apostophes like 'classification' should now be enclosed in`<info>`. This helps us show nicer docs on the web. The tag gets replaced when viewed on the CLI. For example see https://github.com/drush-ops/drush/blob/8fd1118cbb4bff069ade0f1aa89ed9d6df8ba496/src/Commands/core/SiteInstallCommands.php#L29 and https://www.drush.org/commands/10.x/site_install/#arguments.

There are numerous instances of this in this file.